### PR TITLE
Upgrade akka to 2.6.1

### PR DESCRIPTION
**Pull request by kpritam**

### CODE REVIEW

**Comment by RustedBones** [comment]
Scala 2.11 support must be dropped to allow this update

**Comment by kpritam** [comment]
yes, Akka 2.6.x dropped support for 2.11

File: graphite/src/main/scala/fr/davit/akka/http/metrics/graphite/CarbonClient.scala [hunk 1]
```diff
@@ -20,7 +19,7 @@ object CarbonClient {
 
 class CarbonClient(host: String, port: Int)(implicit system: ActorSystem) extends AutoCloseable {
 
-  implicit private lazy val materializer: Materializer = ActorMaterializer()
+  implicit private lazy val materializer: Materializer = Materializer(system)
```
Comment (RustedBones) [inline graphite/src/main/scala/fr/davit/akka/http/metrics/graphite/CarbonClient.scala]: If I check the [akka 2.6 migration guide](https://doc.akka.io/docs/akka/current/project/migration-guide-2.5.x-2.6.x.html#akka-stream-changes), I'm not sure we need to declare a materializer at all now

Comment (kpritam) [reply graphite/src/main/scala/fr/davit/akka/http/metrics/graphite/CarbonClient.scala]: Yes, they now have implicit def which converts actor system to mat
I have corrected it.

**Pull request merged**